cpp/examples/abm.cpp

```diff
@@ -17,7 +17,11 @@
 * See the License for the specific language governing permissions and
 * limitations under the License.
 */
+
+#include "epidemiology/abm/household.h"
 #include "epidemiology/abm/abm.h"
+#include <stdio.h>
```

Comment:
In c++, the header is included as `<cstdio>` instead of `<stdio.h>`, same for all other C standard library headers.

Reply:
fixed